### ISSUE TITLE
Missing tags in examples

### ISSUE THREAD

**Issue by maliMirkec** [GoogleChrome/web.dev#5876]
### What page(s) need to be updated?

https://web.dev/aurora-resource-inlining/

### Why is this update needed?

Although the browser will render examples correctly, I think that it is better to show developers the best practice, especially less experienced developers in this field.

Current example:
```html
<head>
<link rel="stylesheet" href="styles.css" media="print" onload="this.media='all'">
<section>
 <button class="primary"></button>
</section>
```

Better example: 
```html
<head>
<link rel="stylesheet" href="styles.css" media="print" onload="this.media='all'">
...
</head>
<body>
 <section>
  <button class="primary"></button>
 </section>
</body>
```

Current example:
```html
<head>
<link rel="stylesheet" href="styles.css" media="print" onload="this.media='all'">
<style>
section button.primary {
  /* ... */
}
</style>
</head>
<section>
 <button class="primary"></button>
</section>
``

Better example: 
```html
<head>
<link rel="stylesheet" href="styles.css" media="print" onload="this.media='all'">
<style>
section button.primary {
  /* ... */
}
</style>
</head>
<body>
 <section>
  <button class="primary"></button>
 </section>
</body>
```

**Comment by samthor** [comment]
cc @mgechev (not sure this will work, GitHub won't autocomplete the username)

**Comment by mgechev** [comment]
Adding the missing tags makes sense. @maliMirkec would you want to open a PR with the change or should I go ahead and do that?

**Comment by NegiAkash890** [comment]
I can help here?

**Comment by robdodson** [comment]
temporarily assigning to @mgechev while we wait to hear back from @maliMirkec.

@NegiAkash890 thank you for volunteering. I'd like to hear back from @maliMirkec first just so we don't end up with two folks creating the same PR.

**Comment by maliMirkec** [comment]
I'll do it today!

**Comment by maliMirkec** [comment]
@mgechev Here's the pull request: https://github.com/GoogleChrome/web.dev/pull/5886.

**Comment by jpmedley** [comment]
Addressed by #5886
